app/components/deleteButton.tsx

```diff
@@ -1,28 +1,28 @@
-"use client";
+'use client';
 
-import React from "react";
-import { Button } from "@mui/material";
-import DeleteIcon from "@mui/icons-material/Delete";
-import { useTaskContext } from "../context/TaskContext";
+import React from 'react';
+import { Button } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useTaskContext } from '../context/TaskContext';
 
 interface AddButtonProps {
-	taskId: string;
+  taskId: string;
 }
 
 const AddButton: React.FC<AddButtonProps> = ({ taskId }) => {
-	const { deleteTask } = useTaskContext();
+  const { deleteTask } = useTaskContext();
 
-	const handleSubmit = () => {
-		deleteTask(taskId);
-	};
+  const handleSubmit = () => {
+    deleteTask(taskId);
```

Comment:
why don't you directly use deleteTask method instead you're creating a separate method that just then uses the same method

Reply:
At first I wrote some logic in handleSubmit, but now, you are right, I don't need a separate function.

Reply:
At first I had some logic in handleSubmit, but now, you are correct, there is no need in separate function call